nextflow.config

```diff
@@ -10,23 +10,22 @@ plugins {
 nextflow.enable.strict = true
 
 params {
-    // trace_timestamp = new java.util.Date().format('yyyy-MM-dd_HH-mm-ss')
+    singularity_cachedir = "singularity_cache"
```

Comment:
Why do we need a parameter for the cache? This is usually handled by singularity itself (or the environment).
I wouldn't remove it now... but in a next iteration